server/src/main/java/de/uftos/utils/SpecificationBuilder.java

```diff
@@ -64,8 +68,41 @@ public SpecificationBuilder<T> optionalAndJoinIn(Optional<String[]> attributeVal
     }
     specification = specification.and(
         (root, query, cb) -> root.join(relationName).get(attributeName)
-            .in(List.of(attributeValue.get()))
+            .in(Arrays.stream(attributeValue.get()).toList())
     );
     return this;
   }
+
+  public SpecificationBuilder<T> andEqual(String attributeValue, String attribute) {
+    specification =
+        specification.and((root, query, cb) -> cb.equal(root.get(attribute), attributeValue));
+    return this;
+  }
+
+  public SpecificationBuilder<T> optionalAndJoinLike(Optional<String> attributeValue,
```

Comment:
This should also be called `optionalAndJoinLikeIgnoreCase` or `optionalAndJoinILike`

Reply:
@SuperMarcomen if this has been resolved I'm happy to merge it

Reply:
done